### ISSUE TITLE
glareReverse not working

### ISSUE THREAD

**Issue by MelodicCrypter** [mkosir/react-parallax-tilt#15]
### 🐛 Issue/Bug Report Template

**Describe the bug**  
If **glareReverse** is enabled it won't work. The glare would never show. If I change the **glarePosition** to "_bottom_" or "_top_", as long as it is not set to "_all_", the glare would come back but it is **not** reversed still.

**Expected behavior**  
If you hover on the top-right, the glare should be at the bottom areas.

**Code snippets**  
Inside React app...
```
return (
    <div className="wrapper">
        <Tilt glareReverse glareEnable glareMaxOpacity={0.4} glareColor="#ffffff" glarePosition="all">
            <img src="./testImage.jpg" />
        </Tilt>
    </div>
  );
```

Inside the scss file...
```
.wrapper {
  padding: 1px;
  width: 90%;
  height: auto;
  margin: 0 auto;
  border-radius: 10px;

  img {
    width: 100%;
    border-radius: 9px
  }
}
```

**Environment**

- Browser: Chrome
- Version: "react-parallax-tilt": 1.4.66
- ...

**Additional context**

**Comment by mkosir** [comment]
Hi @MelodicCrypter thanks for the bug report.
Bug fix is merged and deployed in the newest version 1.4.74, can you check if it's working for you as expected (or add reproducible demo link)?

**Comment by mkosir** [comment]
Closing this issue since inactivity, feel free to reopen it!
